sensor_msgs/msg/Attitude.msg

```diff
@@ -0,0 +1,14 @@
+# This message represents 3D attitude
+
+int8 ENU=1
+int8 NED=2
+int8 ENU_FLOATING=3
+int8 NED_FLOATING=4
+
+std_msgs/Header header
+
+geometry_msgs/Quaternion orientation
+float64[9] orientation_covariance # Row major about x, y, z axes
+
+uint8 reference_frame_type # NED, ENU, NED_FLOATING, ENU_FLOATING
+string sensor_frame # The coordinate frame in the system of the observation.
```

Comment:
Isn't this duplicating info from header?

Reply:
No. From https://groups.google.com/g/ros-sig-drivers/c/2NvgNBOjcFQ/m/LfG6fhFbIBwJ:

```text
Overview of frames for AHRS

/map # semi-global reference
|
---> ahrs_fix_ned/enu
|
---> odom
       | ---> base_link
                 |------> imu_link (the frame in which the ahrs sensor is mounted, possibly anywhere)
```

The `sensor_frame` should always be the frame of the IMU (`/imu_link` in this example). The way I understood the original proposal, the `header.stamp` should be the reference frame, like this:

* if `reference_frame_type == ENU` then `header.stamp = ahrs_fix_enu`
* if `reference_frame_type == NED` then `header.stamp = ahrs_fix_ned`
* if `reference_frame_type == ENU_FLOATING` then `header.stamp = ahrs_fix_enu_floating`
* if `reference_frame_type == NED_FLOATING` then `header.stamp = ahrs_fix_ned_floating`

... where the TF frames `ahrs_fix_enu_floating` and `ahrs_fix_ned_floating` don't exist (= are not connected to the `/map` frame). This is a bit weird, but the "floating" reference frames mean that the attitude is relative to some arbitrary reference frame (usually the startup pose of the sensor), and the transformation between `/map` and the startup pose is not known.

Reply:
Something along these lines should probably added as a comment to this message.